### PR TITLE
Allow env disabling of auth providers

### PR DESCRIPTION
**Pull request by markphilpot**

### CODE REVIEW

File: src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java [hunk 1]
```diff
@@ -26,12 +26,32 @@
...
+        {
+            @Override
+            public void postStart()
+            {
+                Set<String> disabledProviders = Arrays.stream(
+                        Configuration.getInstance().getString("security.disabledAuthorizationProviders", "").split(","))
+                        .map(String::trim)
+                        .collect(Collectors.toSet());
+                enabledProviders = getProviders().stream()
+                        .filter(p -> !disabledProviders.contains(p.getClass().getName()))
```
Comment (samiur) [inline src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java:43]: Hmmm doesn't this potentially set you up to accidentally disable providers you may not want to? Depending on what `getName()` returns

**Review by samiur — CHANGES_REQUESTED** [review]
minor comment

File: src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java (hunk 1 above)
Comment (markphilpot) [reply src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java:43]: getName returns the fully qualified classname  -- so `com.canopyiq.security.FooBar`

Comment (samiur) [reply src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java:43]: Looking through docs, looks like `getName` provides the FQDN. But, you could still get in this scenario:
```
security.disabledAuthorizationProviders="com.journal.Auth0AuthProviderComplex"

...
getProviders() -> [com.journal.Auth0AuthProvider, com.journal.Auth0AuthProviderComplex]
```

In the above scenario, both of those providers would be disabled, but that might not be intended.

**Pull request merged**